### ISSUE TITLE
data deduplicate

### ISSUE THREAD

**Issue by pawopawo** [Alibaba-MIIL/ImageNet21K#22]
Has the ImageNet1K validation data and Imagenet21K training data been de-duplicated?

**Comment by mrT23** [comment]
the validation set of ImageNet1K  is not in Imagenet21K  train set, i cross-checked that.
so this is a "fair" scenerio

**Comment by pawopawo** [comment]
How can I check if there are duplicates?

**Comment by mrT23** [comment]
i am sure the internet contains guides on that.
I did a loop-for on the validation set files
